config/sync/core.entity_form_display.node.stanford_page.default.yml

```diff
@@ -90,14 +89,10 @@ content:
     region: content
     third_party_settings: {  }
   su_basic_page_type:
-    type: options_shs
+    type: options_select
```

Comment:
Why this change?

Reply:
Ishi found the simple hierarchal select was confusing, in that it appeared to the inexperienced user to apply two or more terms if you choose a term below the root level of the taxonomy.  By changing it to a simple select, it's more obvious that only one term is being applied.

Reply:
i guess that kinda makes sense but could also be resolved with help text. nothing i have a strong opinion about though.